build.ps1

```diff
@@ -77,6 +82,10 @@ function Invoke-Configure {
         choco install habitat_native_dependencies --confirm -s https://www.myget.org/F/habitat/api/v2  --allowemptychecksums
     }
 
+    choco install libzmq_vc120 --version 4.2.3 --confirm -s https://www.nuget.org/api/v2/ --allowemptychecksums
```

Comment:
This is probably ok for now, but everything else (including the rust bits) is binding against the vc140 runtime.